tests/unit/source_code/test_pyspark.py

```diff
@@ -46,6 +46,75 @@ def test_spark_sql_match(migration_index):
     ] == list(sqf.lint(old_code))
 
 
+def test_spark_table_match(migration_index):
+    # import SPARK_MATCHERS from
+    for method_name in ["cacheTable", "createTable", "createExternalTable", "getTable", "isCached",
```

Comment:
this usage hints at `@pytest.mark.parametrize`

Reply:
done